packages/appkit/src/client.ts

```diff
@@ -416,9 +417,18 @@ export class AppKit {
       sdkVersion: SdkVersion
     }
   ) {
+    OptionsController.setDebug(options.debug)
     OptionsController.setProjectId(options.projectId)
     OptionsController.setSdkVersion(options.sdkVersion)
 
+    if (!options.projectId) {
+      AlertController.open('Project ID Not Configured', 'error')
+      // eslint-disable-next-line no-console
+      console.error('Project ID Not Configured - update configuration')
```

Comment:
Should we  use a custom logger for this, also should we have this error in the consts?